tools/sidekick/config.schema.json

```
@@ -0,0 +1,112 @@
+{
+  "$id": "https://ns.adobe.com/helix/sidekick/config",
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "description": "Helix Sidekick configuration",
+  "type": "object",
+  "properties": {
+    "project": {
+      "type": "string",
+      "description": "The name of the project to display in the Helix Sidekick"
+    },
+    "host": {
+      "type": "string",
+      "format": "hostname",
+      "description": "The host name of the production website"
+    },
+    "container": {
+      "type": "string",
+      "description": "The ID of the dropdown container to add this plugin to"
+    },
+    "pushDown": {
+      "type": "boolean",
+      "description": "Have Helix Sidekick push down page content instead of overlaying it"
+    },
+    "pushDownSelector": {
+      "type": "string",
+      "description": "The query selector for absolute elements to also push down"
+    },
+    "plugins": {
+      "type": "array",
+      "items": { "$ref": "#/$defs/plugin" }
+    }
+   },
+   "additionalProperties": false,
+   "$defs": {
+    "plugin": {
+      "type": "object",
+      "properties": {
+        "container": {
+          "type": "string",
+          "description": "The ID of the container to add this plugin to"
+        },
+        "environments": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": ["any", "edit", "preview", "live", "prod"]
+          },
+          "description": "The environments to display this plugin in (default: any)"
```

```suggestion
          "description": "The environments to display this plugin in (default: any)",
          "default": "any"
```

You can make the default explicit.